package.json

```diff
@@ -29,5 +25,8 @@
   "devDependencies": {
     "rollup": "^0.52.1",
     "tape": "^4.3.0"
+  },
+  "dependencies": {
+    "typescript": "^2.7.1"
```

Comment:
typescript should be put in `devDependencies`.

Reply:
miss